tests/sentry/replays/test_organization_replay_index.py

```diff
@@ -287,25 +287,6 @@ def test_get_replays_minimum_field_set(self):
             assert len(response_data["data"][0]) == 1
             assert "id" in response_data["data"][0]
 
-    def test_get_replays_require_timely_initial_sequence(self):
-        """Test returned replays can not partially fall outside of range."""
-        project = self.create_project(teams=[self.team])
-
-        replay1_id = uuid.uuid4().hex
-        replay1_timestamp0 = datetime.datetime.now() - datetime.timedelta(days=365)
-        replay1_timestamp1 = datetime.datetime.now() - datetime.timedelta(seconds=10)
-
-        self.store_replays(mock_replay(replay1_timestamp0, project.id, replay1_id, segment_id=0))
-        self.store_replays(mock_replay(replay1_timestamp1, project.id, replay1_id, segment_id=1))
-
-        with self.feature(REPLAYS_FEATURES):
-            response = self.client.get(self.url)
-            assert response.status_code == 200
-
-            response_data = response.json()
-            assert "data" in response_data
-            assert len(response_data["data"]) == 0
-
```

Comment:
The only breaking change!  As a performance optimization I'm allowing replays which partially fall inside the range to be returned.  This optimization can be removed if it causes issues.  I'm hoping it doesn't though...